components/dockmanager/pane-types.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Pane Types
+page_title: DockManager - Pane Types
+description: Pane Types in the DockManager for Blazor.
+slug: dockmanager-pane-types
+tags: telerik,blazor,dockmanager,pane,types
+published: true
+position: 15
+---
+
+# Pane Types
+
+The Blazor DockManager component exposes the ability to configure different pane types.
+
+## Panes Nested Tags Settings
+
+When defining pane types, the naming convention follows the structure `<DockManager*Type*Pane>`, where **Type** specifies the behavior of the pane. The available types are:
+
+#### DockManagerContentPane
+
+Provides full control over explicitly defining custom content to be rendered for a given pane based on specific requirements. 
+
+* It can be a direct child of all other panes and the `<DockManagerPanes>` tag.
+
+#### DockManagerTabGroupPane
+
+Groups panes in a tab strip, similar to the [TabStrip component](slug://components/tabstrip/overview). Users can navigate through panes using tabs in the header. 
+
+* It can be a direct child of `<DockManagerSplitPane>` and the `<DockManagerPanes>` tag. 
+* It can only contain `<DockManagerContentPane>` children.
+
+#### DockManagerSplitPane
+
+Organizes panes in a [Splitter-like](slug://splitter-overview) manner, allowing the container pane to be split either horizontally or vertically. 
+
+* It can be a direct child of another `<DockManagerSplitPane>` and the `<DockManagerPanes>` tag. 
+* It can contain `<DockManagerTabGroupPane>`, `<DockManagerContentPane>`, and other `<DockManagerSplitPane>` tags as children. 
+* Only this pane type can be declared as a direct child of the `<DockManagerFloatingPanes>` tag.
+
+>caption DockManager with all pane types.
```

Comment:
Currently this example is very similar to the basic one in Overview. It should either provide some unique value, which the other examples don't or otherwise it can be removed for simplicity.